src/addlocal.f

```diff
@@ -1,45 +1,33 @@
+!>    @file
+!>    @brief This subroutine adds a Local Use Section (Section 2) to
+!>    a GRIB2 message.
+!>    @author Gilbert ORG: W/NP11 @date 2000-05-01
+!>    
+
+!>    This routine is used with routines gribcreate(), addlocal(), addfield(),
```

Comment:
The same info in the @brief above needs to be here also. So change this to:

```
!> This subroutine adds a Local Use Section (Section 2) to
!>    a GRIB2 message. This routine is used with routines gribcreate(), addlocal(), addfield(), ...
```

In the doxygen output, this information appears in two different places. So it needs to be both in the file @brief, and as the first sentence of the subroutine documentation. This applies to all the files you doxygenate.